DESCRIPTION

```diff
@@ -10,7 +10,7 @@ Authors@R: c(
     person("Kun", "Ren", role = "aut"),
     person("Alexander", "Rosenstock", role = "aut",
            comment = "AshesITR"),
-    person("Indrajeet", "Patil", role = "aut", email = "patilindrajeet.science@gmail.com",
```

Comment:
weird to me that leaving an unnamed blank argument is preferred to including only named arguments...